lib/live_view/live_view.dart

```diff
@@ -43,14 +45,20 @@ enum ViewType { deadView, liveView }
 class LiveSocket {
   PhoenixSocket create({
     required String url,
-    required Map<String, dynamic>? params,
-    required Map<String, String>? headers,
+    Map<String, dynamic>? params,
```

Comment:
Could we still keep them as required? Because they are required for the connection to work

Reply:
Remove the `?`

Reply:
Yes